src/impl/서울신문.ts

```diff
@@ -8,7 +8,7 @@ export function parse(): Article {
         subtitle: $('.title_sub').text() || undefined,
         content: (() => {
             const content = $('#atic_txt1')[0].cloneNode(true);
-            $('#hnsIframe, #ifrm_photolink, #googleAdTable', content).remove();
+            $('#hnsIframe, #ifrm_photolink, #googleAdTable', '#seoulAdTable', content).remove();
```

Comment:
가릴 요소를 한 번에 넘기면 됩니다.
```suggestion
            $('#hnsIframe, #ifrm_photolink, #googleAdTable, #seoulAdTable', content).remove();
```